File: app/src/main/java/com/mgaetan89/showsrage/activity/ScheduleActivity.java
package com.mgaetan89.showsrage.activity;

import android.support.v4.app.Fragment;

import com.mgaetan89.showsrage.R;
import com.mgaetan89.showsrage.fragment.ScheduleFragment;

public class ScheduleActivity extends BaseActivity {
	@Override
	protected boolean displayHomeAsUp() {
		return false;
	}

	@Override
	protected Fragment getFragment() {
		return new ScheduleFragment();
	}

	@Override
	protected int getSelectedMenuId() {
		return R.id.menu_schedule;
	}

	@Override
	protected int getTitleResourceId() {
		return R.string.schedule;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/activity/ScheduleActivity.kt
package com.mgaetan89.showsrage.activity

import com.mgaetan89.showsrage.R
import com.mgaetan89.showsrage.fragment.ScheduleFragment

class ScheduleActivity : BaseActivity() {
    override fun displayHomeAsUp() = false

    override fun getFragment() = ScheduleFragment()

    override fun getSelectedMenuId() = R.id.menu_schedule

    override fun getTitleResourceId() = R.string.schedule
}
